src/lib/optimizer/strategy/exists_reformulation_rule.cpp

```diff
@@ -152,8 +153,9 @@ void ExistsReformulationRule::apply_to(const std::shared_ptr<AbstractLQPNode>& n
   }
 
   // Build the join node and put it into the LQP in the place of the predicate
-  const auto join_mode =
-      exists_expression->exists_expression_type == ExistsExpressionType::Exists ? JoinMode::Semi : JoinMode::Anti;
+  const auto join_mode = exists_expression->exists_expression_type == ExistsExpressionType::Exists
+                             ? JoinMode::Semi
+                             : JoinMode::AntiNullAsFalse;
```

Comment:
Hard to search on a tablet: Do we already use AntiNullAsTrue somewhere?

Reply:
Nay - the JoinReformulation group will use it, though